File: sdk/typescript/tests/client.test.ts
import { LLMEngineClient } from '../src/index.js';
import assert from 'node:assert';

const client = new LLMEngineClient({ baseUrl: 'http://localhost:8000', apiKey: 'test_key' });
assert.ok(client, 'Client should be instantiated');
console.log('TypeScript SDK tests passed.');

File: sdk/go/tests/client_test.go
package tests

import (
	"testing"
	"github.com/llm-engine/sdk-go"
)

func TestNewClient(t *testing.T) {
	client := llmengine.NewClient(llmengine.Config{
		BaseURL: "http://localhost:8000",
		APIKey:  "my_api_key",
	})
	if client == nil {
		t.Fatal("Expected non-nil client")
	}
}
